x-pack/legacy/plugins/ml/public/application/components/controls/select_interval/index.d.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+export const interval$: BehaviorSubject<{
```

Comment:
same as `showCharts$`, might be better to combine them into a service

Reply:
Noted in meta issue to follow up https://github.com/elastic/kibana/issues/18553